app/javascript/packages/document-capture/components/file-input.jsx

```diff
@@ -285,13 +290,16 @@ function FileInput(props, ref) {
               )}
             </div>
           )}
-          <div className="usa-file-input__box" />
+          <div className="usa-file-input__box">
+            {isValuePending && <SpinnerDots isCentered className="text-base" />}
+          </div>
           <input
             ref={inputRef}
             id={inputId}
             className="usa-file-input__input"
             type="file"
             aria-label={getLabelFromValue(value)}
+            aria-busy={isValuePending}
```

Comment:
Still awaiting some feedback about this particular implementation, since `aria-busy` doesn't seem to be particularly well-supported in iOS at the moment.

Currently leaning toward using a combination of the existing `aria-label` logic (adding a case for the pending state) and `aria-live`.

Related Slack discussion: https://gsa-tts.slack.com/archives/CNCGEHG1G/p1641391939010000

Reply:
Based on continued discussion in the Slack thread linked above, we decided to use the status message element to communicate these state changes to screen readers via `role="status"`.